tachyon/src/main/resources/tachyon.keywords

```diff
@@ -0,0 +1 @@
+cat,copyFromLocal,copyToLocal,count,du,fileinfo,free,getUsedBytes,getCapacityBytes,load,loadMetadata,location,ls,lsr,mkdir,mount,mv,pin,report,request,rm,rmr,setTTL,unsetTTL,tail,touch,unmount,unpin
```

Comment:
I'm not 100% sure but I believe it is required that any file would have the Apache License, even the embedded resource - like [this one](https://github.com/apache/incubator-zeppelin/blob/master/spark/src/main/resources/python/zeppelin_pyspark.py)

Is this something you could accommodate?